index.html

```diff
@@ -0,0 +1,252 @@
+          <p class="body-2 color-dark-text">
+            @ Alexis Hotel Conference Center, Lilongwe, Malawi 
+          </p>
+        </div>
+      </article>
+      <section class="main-program bg-dark color-white">
+        <h2 class="header-3 text-center">Main Program</h2>
+        <div class="divider bg-primary bd-primary my-12"></div>
+        <ul class="program-list">
+          <li class="program-item">
+            <i class="fas fa-microphone"></i>
+            <p class="color-primary program-title">8 Keynote Addresses</p>
+            <p class="program-description">
+              Delivered by high level dignitaries
+            </p>
+          </li>
+          <li class="program-item">
+            <i class="fas fa-people-arrows"></i>
+            <p class="color-primary program-title">6 high level Panels</p>
+            <p class="program-description">
+              Featuring society leaders; gender experts and young women
+            </p>
+          </li>
+          <li class="program-item">
+            <i class="far fa-comment-dots"></i>
+            <p class="color-primary program-title">20+ #CovidVoices</p>
+            <p class="program-description">
+              Stories of how Covid19 has affected women and girls from across Africa
+            </p>
+          </li>
+          <li class="program-item">
+            <i class="fas fa-user-friends"></i>
+            <p class="color-primary program-title">Workshops</p>
+            <p class="program-description">
+              Partner organised Skills Building workshops
+            </p>
+          </li>
+          <li class="program-item">
+            <i class="far fa-flag"></i>
+            <p class="color-primary program-title">Side events</p>
+            <p class="program-description">
+              Partner organised Side events and Satellite events 
+            </p>
+          </li>
+          <li class="program-item">
+            <i class="fas fa-guitar"></i>
+            <p class="color-primary program-title">Concert</p>
+            <p class="program-description">
+              A Concert and various music, dance and poetry performances
+            </p>
+          </li>
+          <li class="program-item">
+            <i class="fas fa-trophy"></i>
+            <p class="color-primary program-title">Annual AGE Awards</p>
+            <p class="program-description">
+              A celebration of individuals and organisations that have championed gender equality.
+            </p>
+          </li>
+        </ul>
+        <div class="d-flex justify-center my-12">
+          <button type="button" class="join-button button bg-primary">
+            Join AGEConf 2021
+          </button>
+        </div>
+      </section>
+      <section class="featured-speakers bg-white mb-12">
+        <h2 class="header-3 text-center color-secondary">Featured Speakers</h2>
+        <div class="divider bg-primary bd-primary my-12"></div>
+        <ul class="speakers-list"></ul>
```

Comment:
It is nice to see that you've generated the speakers' list with JS :+1: 
It is also a good idea to use `<list>` for each of the speaker items.

- [ ] However, I strongly suggest you use the `<article>` tag for the content of the list items (each speaker). This will be nice for semantic.

Reply:
@teekaytech using an `<article>` tags gives a linter warning of 

> Warning  Article lacks heading. Consider using “h2”-“h6” elements to add identifying headings to all 
> articles.  html-checker

Structurally it's impossible to put a h2 or h6 element in the list item

Reply:
@usmansbk one way of fixing that is to remove the `<ul>` and create articles for each of the speakers, and append each of the articles to the section :blush:

Reply:
@teekaytech that doesn't remove the warning either. The article tag requires a header element to immediately follow it's opening tag, but in this design the first element that follows the opening tag is a speaker image.

What the linter expects is this 
```html
<article>
  <h5>Speaker name</h5>
```

What the design gave is this
```html
<article>
   <image />
```

Reply:
@usmansbk I don't think it matters if the header element comes directly after the opening `<article>` tag. You can use the `hx` tag with the speaker's name. As long as there's a header in the article, I think the linter should pass, no matter if it comes immediately after the opening tag or not.

Please let me know how it turns out :blush: